src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -256,15 +256,17 @@ def apply_tropomi_operator(
         time = pd.to_datetime(str(TROPOMI["utctime"][iSat]))
         strdate = time.round("60min").strftime("%Y%m%d_%H")
         GEOSCHEM = all_date_gc[strdate]
+        dlon = np.median(np.diff(GEOSCHEM["lon"])) # GEOS-Chem lon resolution
+        dlat = np.median(np.diff(GEOSCHEM["lat"])) # GEOS-Chem lon resolution
 
         # Find GEOS-Chem lats & lons closest to the corners of the TROPOMI pixel
         longitude_bounds = TROPOMI["longitude_bounds"][iSat, jSat, :]
         latitude_bounds = TROPOMI["latitude_bounds"][iSat, jSat, :]
         corners_lon_index = []
         corners_lat_index = []
         for l in range(4):
-            iGC = nearest_loc(longitude_bounds[l], GEOSCHEM["lon"])
-            jGC = nearest_loc(latitude_bounds[l], GEOSCHEM["lat"])
+            iGC = nearest_loc(longitude_bounds[l], GEOSCHEM["lon"], tolerance=dlon)
+            jGC = nearest_loc(latitude_bounds[l], GEOSCHEM["lat"], tolerance=dlat)
```

Comment:
If we made this change, the tolerance (which used to default to 0.5) would now be dlon=0.3125 and dlat=0.25 for our typical inversions. I don't know if this will catch everything (e.g., if you have a large TROPOMI pixel, missing GEOS-Chem pixels in-between the TROPOMI pixel corners). Maybe we should just change the tolerance to always 5? Or max(dlon and 1) and max(dlat and 1)? @djvaron 

I will also need to change this in the other TROPOMI operator.